frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts

```diff
@@ -70,14 +62,14 @@ export const getDefaultUISchemaForPropertyName = (name) =>
 
 // Determine if a schema will produce an empty form field.
 export const hasNoFields = (jsonSchema: JSONSchema6): boolean => {
-  const type = getSchemaType(jsonSchema ?? {});
+  const type = getSchemaType(jsonSchema ?? {}) ?? '';
   const handleArray = () => {
     return hasNoFields(jsonSchema.items as JSONSchema6);
   };
   const handleObject = () => {
     return (
       _.every(jsonSchema?.properties, hasNoFields) &&
-      _.every(jsonSchema?.additionalProperties as JSONSchema6, hasNoFields)
+      hasNoFields(jsonSchema?.additionalProperties as JSONSchema6)
```

Comment:
`additionalProperties` is either boolean or JSONSchema6. It doesn't have nested schemas like 'properties'.